pypesto/optimize/ess/ess.py

```diff
@@ -560,7 +560,7 @@ def _do_local_search(
     def _maybe_update_global_best(self, x, fx):
         """Update the global best value if the provided value is better."""
         if fx < self.fx_best:
-            self.x_best = x[:]
+            self.x_best[:] = x
```

Comment:
how is this related to the go-beyond algorithm?

Reply:
It's not. But it's another potential copying issue.